tests/playwright/src/specs/podman-machine-rootless.spec.ts

```diff
@@ -36,7 +41,6 @@ test.afterAll(async ({ runner }) => {
   await runner.close();
 });
 
-test.skip(os.platform() === 'linux', 'Runs only on Windows and Mac');
```

Comment:
Why it's not skipped ?

Reply:
it is, the skip command was moved higher in the file to line 28.